model/src/cfd.rs

```diff
@@ -266,19 +263,13 @@ impl Order {
     ) -> Self {
         let leverage_choices_for_taker = Leverage::TWO;
 
-        let liquidation_price = match position_maker {
-            Position::Short => calculate_short_liquidation_price(leverage_choices_for_taker, price),
-            Position::Long => calculate_long_liquidation_price(leverage_choices_for_taker, price),
-        };
-
         Order {
             id: OrderId::default(),
             price,
             min_quantity,
             max_quantity,
             leverage_taker: leverage_choices_for_taker,
             trading_pair: TradingPair::BtcUsd,
-            liquidation_price,
```

Comment:
❓ Is this a breaking change for our API or database?

Reply:
Since we changed the wire to send `MakerOffer` the `Order` is purely internal. 
We also don't safe the `Order` into the database anymore (the complete table was removed from the database on both sides).

Reply:
Yep, this is all correct AFAIK. No relevant breaking changes here!

Reply:
The automation does not use it but it might still need that field for deserialization. So better double check that it does not break anything.